.github/workflows/release.yml

```diff
@@ -62,6 +75,18 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+    - name: Upload - AEPEdgeIdentity
+      uses: actions/upload-release-asset@v1
+      if: ${{ github.event.inputs.action_tag == 'yes' }}
+      with:
+        upload_url: ${{ steps.create_release.outputs.upload_url }}
+        asset_path: ./build/AEPEdgeIdentity.xcframework.zip
+        asset_name: AEPEdgeIdentity.xcframework.zip
```

Comment:
also include the version in the artifact name:
asset_name: AEPEdgeIdentity-${{ github.event.inputs.tag }}.xcframework.zip